frontend/src/components/PurpleButton.tsx

```diff
@@ -47,12 +49,14 @@ const PurpleButton = ({
   onClick,
   icon,
   check,
+  padding,
```

Comment:
I think if you use a `styled-component` you can add padding there!

Reply:
ya actually that's true. Also we were thinking that maybe in the future if we have other buttons with longer words, then we might need this padding to adjust the purple area